mps_youtube/commands/search.py

```diff
@@ -151,14 +150,40 @@ def channelfromname(user):
     return (user, channel_id)
 
 
+@command(r'channels\s+(.+)')
+def channelsearch(q_user):
+
+    qs = {'part': 'id,snippet',
+          'q': q_user,
+          'maxResults': 50,
+          'type': 'channel',
+          'order': "relevance"
+          }
+
+    QueryObj = contentquery.ContentQuery(listview.ListUser, 'search', qs)
+    columns = [
+        {"name": "idx", "size": 3, "heading": "Num"},
+        {"name": "name", "size": 30, "heading": "Username"},
+        {"name": "description", "size": "remaining", "heading": "Description"},
+        ]
+
+    def run_m(user_id):
+        """ Search ! """
+        usersearch_id(*user_id)
+    del g.content
```

Comment:
It isn't. I found a bug in the code where a second channel search would still yield the results from the first search. I wasn't entirely sure what caused this, so I added some code to make sure i deleted previous data before rewriting them, . When I found the bug, i removed much of that code, seems I skipped this line.